prometheus-jmx-exporter/Dockerfile

```diff
@@ -34,3 +39,17 @@ COPY collect-all-slow.yml example_configs/
 
 ENTRYPOINT ["java", "-jar", "jmx_prometheus_httpserver.jar"]
 CMD ["5556", "example_configs/collect-all-slow.yml"]
+
+FROM base as javaagent
+
+RUN set -ex; \
+  cd ./jmx_exporter; \
+  find jmx_prometheus_javaagent/ -name jmx_prometheus_javaagent-*.jar -exec mv -v '{}' ../jmx_prometheus_javaagent.jar \;; \
+  mv example_configs ../; \
+  cd ..; \
+  \
+  rm -Rf ./jmx_exporter ./maven /root/.m2; \
+  \
+  apt-get purge -y --auto-remove $buildDeps; \
+  rm -rf /var/lib/apt/lists/*; \
+  rm -rf /var/log/dpkg.log /var/log/alternatives.log /var/log/apt
```

Comment:
This isn't exactly a diff, just copy-pasta from above with the appropriate selection in `find`.